app/submitter/convert_payload_0_0_3.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Dict, Optional
+
+from app.data_model.answer import Answer
+from app.data_model.answer_store import AnswerStore
+
+
+def convert_answers_to_payload_0_0_3(answer_store, list_store, schema, routing_path) -> List[Dict]:
+    """
+    Convert answers into the data format below
+    'data': [
+        {
+            'value': 'Joe',
+            'answer_id': 'first-name',
+            'list_item_id': 'axkkdh'
+        },
+        {
+            'value': 'Joe',
+            'answer_id': 'first-name',
+            'list_item_id': 'axkkdh'
+        },
+        {
+            'value': 'No',
+            'answer_id': 'do-you-live-here'
+        }
+    ]
+
+    For list answers, this method will query the list store and get all answers from the
+    add list item block. If there are multiple list collectors for one list, they will have
+    the same answer_ids, and will not be duplicated.
+
+    Returns:
+        A list of answer dictionaries.
+    """
+    answers = AnswerStore()
+
+    for location in routing_path:
+        if schema.get_block(location.block_id)['type'] == 'ListCollector':
+            for answer in get_answers_for_add_block(answer_store, list_store, schema, location):
```

Comment:
Just me being picky but can we make it consistent between calling a method directly here vs calling a reference like on line 44.